apps/storybook/src/LeadersTabbedCard.stories.tsx

```diff
@@ -14,11 +14,34 @@ export const Normal = () => (
       ...Array.from(
         { length: number('Number of leaders', 2, { max: 4 }) },
         (_, index) => ({
-          user: createUserResponse({}, index),
+          user: {
+            ...createUserResponse({}, index),
+            alumniSinceDate: index % 2 === 0 ? '2021-01-01' : undefined,
+          },
           role: 'Lead PI - Chair' as GroupRole,
         }),
       ),
     ]}
     title={text('Title', 'Interest Group Teams')}
+    disableActiveTab={false}
+  />
+);
+
+export const ActiveTabDisabled = () => (
```

Comment:
Could this not be a simple boolean knob in the normal case

Reply:
The other less desirable solution is not to copy paste but take the data out

Reply:
Didn't know about that knob stuff! A new world unfolds, thank you.